Snabble/UI/PaymentMethods/PaymentMethods+UI.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  PaymentMethods+UI.swift
+//
+//  Copyright © 2020 snabble. All rights reserved.
+//
+
+import UIKit
+
+extension PaymentMethodDetail {
+    var icon: UIImage? {
+        switch self.methodData {
+        case .tegutEmployeeCard:
+            return UIImage.fromBundle("SnabbleSDK/payment/payment-tegut")
+        default:
+            return self.rawMethod.icon
+        }
+    }
+}
+
+extension RawPaymentMethod {
+    public var displayName: String {
+        switch self {
+        case .deDirectDebit:
+            return "SEPA-Lastschrift"
```

Comment:
Ist `SEPA-Lastschrift` auch in anderen Sprachen in Ordnung?

Reply:
Ja, das Zahlverfahren heisst so und wird nur in Deutschland angeboten.